modules/RMTL.py

```diff
@@ -0,0 +1,101 @@
+import logging.config
+import modules.commons as commons
+import json
+
+logger = logging.getLogger(__name__)
+
+def readRMTLCSV(path):
+    return  commons.readCSVToDict(path)
+
+def export(targetList,targetsFileName,path):
+    # path correction
+    if not path.endswith("/"):
+        path = path + "/"
+    commons.createDirectoryIfNotfound(path)
+
+    #Save new target files
+    with open(path+targetsFileName, mode="w") as outFile:
+        for target in targetList:
+            outFile.write(json.dumps(target, ensure_ascii=False) + "\n")
+    logger.info("Save Processed Target file to %s", path+targetsFileName)
+
+'''
+Add rmtl propery to the target if match a record in the RMTL list
+
+@:param 
+    target    
+    RMTLList
+@:return target
+'''
+def addRMTLProperyToTarget(target,RMTLList):
+     for rmtl in RMTLList:
+         #  target id matchs Ensembl_ID
+         if rmtl[0]==target["id"]:
+             logger.info("target -- %s --- matches RMTL list %s ", target["id"], rmtl[2])
+             target["rmtl_approved_symbol"]=rmtl[1]
```

Comment:
rmtl_approved_symbol value probably not needed in the JSON, unless you want to keep it for a validation mechanism. 
It is the same as the existing Open Targets ApprovedSymbol value, and both are pulled using the Ensembl_ID. I kept the rmtl_approved_symbol field in the .csv only for validation